cdap-gateway/src/main/java/io/cdap/cdap/gateway/router/RouterPathLookup.java

```diff
@@ -225,9 +225,8 @@ private RouteDestination getV3RoutingService(String[] uriParts, AllowedMethod re
           return LOG_SAVER;
         case Constants.Service.TRANSACTION:
           return TRANSACTION;
-        case Constants.Service.METRICS_PROCESSOR:
-          return METRICS_PROCESSOR;
         case Constants.Service.METRICS:
+        case Constants.Service.METRICS_PROCESSOR:
```

Comment:
Why is this needed?

Is `v3/system/services/status` doesn't display status for `metrics.processor` service, we should not allow `v3/system/services/metrics.processor/status`.